apps/artwork/components/artist/query.coffee

```diff
@@ -36,14 +36,11 @@ module.exports = """
         }
       }
     }
-    exhibition_history: shows {
```

Comment:
It didn't look like this was used anywhere (perhaps it was awaiting this section?).